### DISCUSSION TITLE
Tauri 2.0 SvelteKit - Skeleton Configuration

### DISCUSSION THREAD

**Discussion by 3goats** [skeletonlabs/skeleton#2743] [Q&A]
Hi, I'm using Tauri 2.0 with SvelteKit. I followed the guide here https://www.skeleton.dev/docs/tauri, however I think it's for Tauri 1. Is there any guidance as to how I could install Skeleton into an existing Tauri 2.0 with SvelteKit project ?

**Comment by endigo9740** [comment]
Hey @3goats we don't officially maintain the integration guides for Skeleton v2. The couple we hosted on the doc site are done so as a courtesy to the community, but are very much provided "as is" by their contributors. Per the Tauri guide, it was [contributed by](https://www.skeleton.dev/docs/tauri#attribution) @SebasF1349, so it'll be up to him (or any other community volunteer) if this gets updated.

That said, it looks like Tauri v2 is [currently in beta status](https://v2.tauri.app/blog/tauri-2-0-0-beta/). So it may be wise to wait until they hit a stable release, then provide an update geared towards the upcoming Skeleton v3 update.

**Reply by 3goats** [reply]
Thanks for the response. That makes sense.

**Comment by SebasF1349** [comment]
As endigo9740 mentioned, I will probably not update the guide until Tauri v2 is out of beta. But at least two people have tried integrationg Skeleton with Tauti v2 in our discord server and afaik they didn't have problems. Maybe the instructions needs some tweaks, but if you follow the Tauri guide to integrate with Sveltekit and then the Skeleton manual installation it should work.
